src/components/ClaimItem/ClaimItem.tsx

```diff
@@ -0,0 +1,153 @@
+  claim,
+  onDeleteClaim,
+  onFeedbackSourceRelevance,
+  onFeedbackSourceSupport,
+}) => {
+  // several conditionals when to show what
+  const showIsBeingChecked = claim.isBeingChecked;
+  const showSupportIndicator =
+    claim.hasBeenChecked &&
+    claim.score != undefined &&
+    claim?.evidence &&
+    claim.evidence.length > 0;
+  const showSourcesButton =
+    claim.hasBeenChecked && claim?.evidence && claim.evidence.length > 0;
+  const showSourcesList =
+    claim.hasBeenChecked && claim?.evidence && claim.evidence.length > 0;
+  const showInsufficientData =
+    claim.hasBeenChecked &&
+    !showIsBeingChecked &&
+    !showSupportIndicator &&
+    !showSourcesButton &&
+    !showSourcesList;
+
+  const containsNumbers = /\d/.test(claim.claim);
+
+  let supporting = 0;
+  let balanced = 0;
+  let disputing = 0;
+
+  const [filter, setFilter] = React.useState([
+    { label: 'Supporting', selected: false },
+    { label: 'Disputing', selected: false },
+    { label: 'Neutral', selected: false },
+    { label: 'Google', selected: true },
+    { label: 'Wikipedia', selected: true },
+    { label: 'Google Claim', selected: true },
+    { label: 'Bing News', selected: true },
+  ]);
+  const updateFilter = (newFilter: Array<Filter>) => {
+    setFilter(newFilter);
+  };
+
+  claim.evidence?.forEach((source) => {
+    if (
+      typeof source === 'undefined' ||
+      typeof source.softmaxScore === 'undefined'
+    )
+      return;
+    if (source.softmaxScore[1] < 0.4) disputing++;
+    if (source.softmaxScore[1] >= 0.4 && source.softmaxScore[1] < 0.6)
+      balanced++;
+    if (source.softmaxScore[1] >= 0.6) supporting++;
+  });
+
+  return (
+    <Paper className="ClaimItem" elevation={4}>
+      <Typography component={'span'} align="center">
+        <Grid container direction="column" spacing={2}>
+          <Grid item>
+            <Grid container justifyContent="center" alignItems="center">
+              <Grid item xs={1}></Grid>
+              <Grid item xs={10}>
+                <Typography variant="subtitle1">CLAIM:</Typography>
+              </Grid>
+              <Grid item xs={1}>
+                <Tooltip title="Remove claim" arrow>
+                  <IconButton size="small" onClick={() => onDeleteClaim(claim)}>
+                    <ClearIcon />
+                  </IconButton>
+                </Tooltip>
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid item>
+            <Typography variant="h4">{claim.claim}</Typography>
+          </Grid>
+          {containsNumbers && (
+            <Grid item>
+              <Typography variant="subtitle1">
+                (Our technology currently works best with textual claims.
+                Results could be inaccurate if you have numbers in the claim)
+              </Typography>
+            </Grid>
+          )}
+          {showSupportIndicator && (
+            <Grid item>
+              <FilterBar
```

Comment:
Can you have a look why the color of SupportIcon is different here? In SupportIcon it works correctly and gets the right color from the theme, but here it looks too dark.
<img width="528" alt="Screenshot 2022-08-19 at 14 01 57" src="https://user-images.githubusercontent.com/44401648/185614288-51ba9ca3-fe59-4586-9500-5b1e1a518c32.png">
<img width="1064" alt="Screenshot 2022-08-19 at 13 59 07" src="https://user-images.githubusercontent.com/44401648/185614123-b3f7383d-0d91-4936-ab6a-4b3086ca9d34.png">

Also adjust the colors in preview.js while you are at it, we wanted to go back to the lighter green and red:
```
palette: {
    primary: {
      main: '#ffffff',
    },
    secondary: {
      main: '#f8c608',
    },
    error: {
      main: '#ff8761',
    },
    warning: {
      main: '#fecd0c',
    },
    success: {
      main: '#86c351',
    },
  },
...

```

Reply:
Unfortunately I have no idea, looked into it and indeed it should be a lighter color, but something overlay it somehow
<img width="1281" alt="image" src="https://user-images.githubusercontent.com/38763304/187097789-c9a41a34-631f-4740-89ad-bdd8d6ccbccd.png">

Reply:
I've turned this into an issue, I can't find the problem either, probably something with the theming.